packages/@o3r/core/schematics/rule-factories/additional-modules/index.ts

```diff
@@ -119,8 +119,17 @@ export function updateAdditionalModules(options: { projectName: string | null },
     }
 
     const workspaceProject = getProjectFromTree(tree);
-    const envDevFilePath = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.development.ts');
-    if (!envDevFilePath) {
+    const envProdFilePathNg14 = path.join(path.dirname(workspaceProject.architect!.build.options.main), 'environments', 'environment.prod.ts');
```

Comment:
same comment as below

Reply:
updated. we'll base on env dev file name